forge-game/src/main/java/forge/game/ability/effects/ChangeZoneAllEffect.java

```diff
@@ -107,10 +107,8 @@ public void resolve(SpellAbility sa) {
             final Zone originZone = game.getZoneOf(c);
 
             // Fizzle spells so that they are removed from stack (e.g. Summary Dismissal)
-            if (sa.hasParam("Fizzle")) {
```

Comment:
@tool4ever can we look to remove other Fizzle Params from the ChangeZone Effects?

Reply:
should be fixed